src/main.go

```diff
@@ -2,7 +2,7 @@
 package main
 
 import (
-	dbManager "json9512/mediumclone-go/db"
+	dbManager "github.com/json9512/mediumclone-backendwithgo/src/db"
```

Comment:
aliasing은 진짜 필요한 경우에만 하면 좋을듯 

```suggestion
"github.com/json9512/mediumclone-backendwithgo/src/db"
```

Reply:
```go
    nested "github.com/antonfisher/nested-logrus-formatter"
```
바로 아래 `nested` 으로 aliasing 된 거는 괜찮은건가요?

```go
     // line 28: 사용시
     log.SetFormatter(&nested.Formatter{ ... })
```

Reply:
네네 보통 그 패키지에서 이렇게 사용하면 좋다고 하면 그렇게 하면 됩니당. aliasing은 해도 되고 안해도 되고. 길거나 중복될때 하면 좋은듯여